allennlp/predictors/bidaf.py

```diff
@@ -38,3 +44,101 @@ def _json_to_instance(self, json_dict: JsonDict) -> Instance:
         question_text = json_dict["question"]
         passage_text = json_dict["passage"]
         return self._dataset_reader.text_to_instance(question_text, passage_text)
+
+    @overrides
+    def predictions_to_labeled_instances(self,
+                                         instance: Instance,
+                                         outputs: Dict[str, np.ndarray]) -> List[Instance]:
+        """
+        NAQANET has the following fields, answer_as_passage_spans,  answer_as_question_spans,
+        answer_as_add_sub_expressions, answer_as_counts. We need to provide labels for all of them.
+        """
+        new_instance = deepcopy(instance)
+        # For BiDAF
+        if 'best_span' in outputs:
+            span_start_label = outputs['best_span'][0]
+            span_end_label = outputs['best_span'][1]
+            new_instance.add_field('span_start',
```

Comment:
Add a `passage_field: SequenceField = new_instance['passage']  # type: ignore` above this line, so you can replace `new_instance['passage']` with `passage_field` and remove the `type: ignore` (on both calls).  This should also make it fit on one line.